google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -313,6 +313,40 @@
     ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 """
 
+CREATE_GKE_TABLE = """
+    CREATE TABLE `{0}` (
+        `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+        `project_id` varchar(255) DEFAULT NULL,
+        `addons_config` json DEFAULT NULL,
+        `cluster_ipv4_cidr` varchar(255) DEFAULT NULL,
+        `create_time` datetime DEFAULT NULL,
+        `current_master_version` varchar(255) DEFAULT NULL,
+        `current_node_count` int DEFAULT NULL,
+        `current_node_version` varchar(255) DEFAULT NULL,
+        `endpoint` varchar(255) DEFAULT NULL,
+        `initial_cluster_version` varchar(255) DEFAULT NULL,
+        `instance_group_urls` varchar(255) DEFAULT NULL,
+        `legacy_abac` json DEFAULT NULL,
+        `locations` varchar(255) DEFAULT NULL,
+        `logging_service` varchar(255) DEFAULT NULL,
+        `master_auth` json DEFAULT NULL,
+        `monitoring_service` varchar(255) DEFAULT NULL,
+        `name` varchar(255) DEFAULT NULL,
+        `network` varchar(255) DEFAULT NULL,
+        `node_config` json DEFAULT NULL,
+        `node_ipv4_cidr_size` int DEFAULT NULL,
+        `node_pools` varchar(4096) DEFAULT NULL,
```

Comment:
node_pools is a repeated field that can be much larger than 4k. This should be a json field, not a varchar.

Reply:
Done.